source/_patterns/03-organisms/accordion/_accordion.twig

```diff
@@ -14,6 +14,8 @@
   * https://getbootstrap.com/docs/4.0/components/card/
 #}
 
+{% set accordion_id = random(9999) %}
```

Comment:
Use `random()` with no args for now. I want to get feedback on the screenreader experience here first. Later we can set a global var to seed this, ie something like `random(ACCESSIBILITY_ID_MAX)` set in PL, Drupal, and Grav.